docs/Manpage.md

```diff
@@ -747,6 +747,9 @@ Fetch the newest version of Homebrew and all formulae from GitHub using `git`(1)
 * `-f`, `--force`:
   Always do a slower, full update check (even if unnecessary).
 
+If `HOMEBREW_NO_UPDATE_REPORT_FORMULAE`/`HOMEBREW_NO_UPDATE_REPORT_CASKS` is set, `brew update` will not show the list of newly added formulae/casks, respectively.
```

Comment:
Define this in `env_config.rb` instead.

Reply:
`boolean:     true` there means variable type, not default value, correct?

Reply:
moved it to env_config